assets/src/dashboard/utils/useTemplateView.js

```diff
@@ -19,17 +19,16 @@
  */
 import { useCallback, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
-
 /**
  * Internal dependencies
  */
-import { clamp } from '../../animation';
 import {
   TEMPLATES_GALLERY_SORT_OPTIONS,
   TEMPLATES_GALLERY_STATUS,
   VIEW_STYLE,
 } from '../constants';
 import { PageSizePropType } from '../types';
+import clamp from '../../utils/clamp';
```

Comment:
That import looks incorrect. Shouldn't that be

```suggestion
import clamp from './clamp';
```